projects/05-fleet-management-api/README.md

```diff
@@ -210,41 +210,135 @@ Puedes implementar este proyecto en JavaScript, Python o Java.
 
 ## 7. Pistas, tips y lecturas complementarias
 
-### Modelamiento de datos
+Te proponemos los siguientes pasos para iniciar con el proyecto
+
+### Paso 1. Comprender que es una API REST
+
+En primer lugar, asegurate de comprender qué es una API REST.
+Para esto puedes consultar en internet o preguntarle a
+ChatGpt. Agenda una OH con una coach para confirmar tus aprendizajes.
+En particular, te recomendamos ver este
+[video](https://www.youtube.com/watch?v=1hpc70_OoAg&t=15197s)
```

Comment:
Apoyaria eso con una explanacion en texto 
https://dev.to/dennysjmarquez/todo-lo-que-necesitas-saber-sobre-api-rest-glosario-de-terminos-esenciales-y-mas-29pc

y esto en portugues https://www.freecodecamp.org/portuguese/news/tutorial-de-apis-rest-client-rest-servicos-rest-e-chamadas-de-api-explicados-com-exemplos-de-codigo/

Reply:
Ivy me parece mejor dejar unicamente el articulo que recomiendas. El video que yo recomende es de NodeJS entonces puede confundir a las estudiante que no estan con esa tecnologia.